content/ja/guides/configuration-glossary/configuration-srcdir.md

```diff
@@ -47,7 +47,7 @@ export default {
 
 例 2:
 
-例 1 の代わりに nuxt.config を `src` フォルダに移動することもできます。この場合 client を `rootDir` として指定するだけでよく、`srcDir` は空にできます:
+例 1 の代わりに nuxt.config を `client` フォルダに移動することもできます。この場合 client を `rootDir` として指定するだけでよく、`srcDir` は空にできます:
```

Comment:
```suggestion
例 1 の代わりに nuxt.config を `client` フォルダに移動することもできます。この場合 `client` を `rootDir` として指定するだけでよく、`srcDir` は空にできます:
```

Could you enclose "client" in backticks?

rel: https://github.com/nuxt/nuxtjs.org/commit/57bb31b50c322a4f58052c5aa3199d15ff44d152

Reply:
@aytdm Thank you for review! 

> Could you enclose "client" in backticks?

I fixed it in 9ff39ac86510c56cf658e9ed8fc27b60a5e0bcb3 .